README.md

```diff
@@ -105,388 +73,563 @@ Localhost is fine for development, but
-Then I see a link to sort shelters in alphabetical order
-When I click on the link
-I'm taken back to the Shelters Index Page where I see all of the shelters in alphabetical order
+When I create a review for a shelter
+And do not fill in the field for an image
+A default image is used and displayed for that review upon submission
 ```
+
+```
+[ ] done
+
+User Story 39, Sortable Reviews
+
+As a visitor,
+When I visit a shelter's show page to see their reviews,
+I see additional links to sort their reviews in the following ways:
+- sort reviews by highest rating, then by descending date
+- sort reviews by lowest rating, then by ascending date
+```
+
+```
+[ ] done
+
+User Story 40, More Shelter Statistics
+
+As a visitor,
+When I visit the shelter's index page
+I see the top 3 highest rated shelters highlighted on a specific part of the page
+```
+
+
+
+
+## Rubric
+Note: In order to get 4's criteria under 4's must be completed.
+
+| | **Feature Completeness** | **Rails** | **ActiveRecord** | **Testing and Debugging** |
+| --- | --- | --- | --- | --- |
+| **4: Exceptional**  | All User Stories 100% complete including all sad paths and edge cases, and some extension work completed | Students implement strategies not discussed in class to effectively organize code using POROs and adhere to MVC | Highly effective and efficient use of ActiveRecord beyond what we've taught in class. Even `.each` calls will not cause additional database lookups. | Very clear Test Driven Development. Test files are extremely well organized and nested. Students utilize `before :each` blocks. 100% coverage for features and models |
+| **3: Passing** | Students complete all User Stories. No more than 2 Stories fail to correctly implement sad path and edge case functionality. | Students use the principles of MVC and POROs vs. Models to effectively organize code. Students can defend any of their design decisions. Action View helpers are used. Routes and Actions are following RESTful conventions. | ActiveRecord is used in a clear and effective way to read/write data using no Ruby to process data. Project fully leverages AR associations and methods | 100% coverage for models. 98% coverage for features. Tests are well written and meaningful. Scored a 3 or higher in Feature Completeness |
+| **2: Passing with Concerns** | Students complete all but 1 - 3 User Stories | Students utilize MVC and POROs to organize code, but cannot defend some of their design decisions. Some routes and actions are not restful. | Ruby is used to process data that could use ActiveRecord instead. | Feature test coverage between 90% and 98%, or model test coverage below 100%, or tests are not meaningfully written or have an unclear objective. |
+| **1: Failing** | Students fail to complete 4 or more User Stories | Students do not effectively organize code using MVC and/or POROs | Ruby is used to process data more often than ActiveRecord | Below 90% coverage for either features or models. |  
```

Comment:
Updated the README to include the paired project requirements instead of the solo requirements. Figured it might be easier to have that stored here instead of having to open a separate window to see the instructions. I know we will have to update this at the end of our project, but for now I think we should have the instructions.